azkaban-common/src/main/java/azkaban/executor/ExecutorApiGateway.java

```diff
@@ -75,7 +114,10 @@ public ExecutorApiGateway(final ExecutorApiClient apiClient) {
           .valueOf(executionId)));
       paramList.add(new Pair<>(ConnectorParams.USER_PARAM, user));
 
-      return callForJsonObjectMap(host, port, "/executor", paramList);
```

Comment:
This change causes an exception in non-containerized web server:
```
2021/02/01 20:36:31.053 +0000  INFO [ExecutorHealthChecker] [azk-health-checker] [Azkaban] Begin executor healthcheck routine.
2021/02/01 20:36:31.064 +0000 ERROR [RestfulApiClient] [azk-health-checker] [Azkaban] Unable to parse response as the response status is 404
2021/02/01 20:36:31.065 +0000 ERROR [ExecutorHealthChecker] [azk-health-checker] [Azkaban] Failed to get update from executor - executor-id: 21, executor-host: zeus000279-481057.azure.linkedin.com, executor-port: 11222
azkaban.executor.ExecutorManagerException: <html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1"/>
<title>Error 404 NOT_FOUND</title>
</head>
<body><h2>HTTP ERROR 404</h2>
<p>Problem accessing /. Reason:
<pre>    NOT_FOUND</pre></p><hr /><i><small>Powered by Jetty://</small></i><br/>
...
        at azkaban.executor.ExecutorApiGateway.callWithExecutionId(ExecutorApiGateway.java:122)
        at azkaban.executor.ExecutorHealthChecker.checkExecutorHealth(ExecutorHealthChecker.java:140)
        at azkaban.executor.ExecutorHealthChecker.checkExecutorHealthQuietly(ExecutorHealthChecker.java:104)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
        at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
```
This is because `checkExecutorHealth` calls that API without an execution id.